template/builtin_binary.inc.tmpl

```diff
@@ -6,7 +6,9 @@
 %   ary = RubyVM.enum_for(:each_builtin).to_a
 %   ary.each{|feature, iseq|
 
-static const unsigned char <%= feature %>_bin[] = {
+#include "compile.h"
+
+static const unsigned char <%= feature %>_bin[] __attribute__((aligned(alignof(struct ibf_header)))) = {
```

Comment:
This is non-portable clearly.

Reply:
It works with GCC and Clang. It doesn't with MSVC, I guess I can come up with a way that will also work with MSVC. Is there any other compiler I need to be aware of?